test/validate-prowgen-breaking-changes.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+workdir="$( mktemp -d )"
+trap 'rm -rf "${workdir}"' EXIT
+
+git clone https://github.com/openshift/release.git --depth 1 "${workdir}/release"
```

Comment:
Can we make this optional and turn it off in CI since `clonerefs` does this for us?

Reply:
Hmm this should be run in presubmit for `openshift/ci-operator-prowgen` but needs to check out `openshift/release`. Is this possible to set up a presubmit like that? Somehow with `extra_refs`?

Reply:
Yeah, totally - just add `openshift/release@master` to `extra_refs`

Reply:
That won't work for `ci-operator` tests.

Reply:
:man_facepalming:

Reply:
It was also what I thought when I saw that line ;)  Too bad it doesn't work.